src/main/java/lotto/domain/LottoNumber.java

```diff
@@ -0,0 +1,63 @@
+package lotto.domain;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Objects;
+import java.util.stream.IntStream;
+
+import lotto.exception.LottoNumberValidationException;
+
+public class LottoNumber implements Comparable<LottoNumber>{
+
+	private static Map<Integer, LottoNumber> lottoNumberMap = new HashMap<>();
+	private final int lottoNumber;
+
+	static {
+		IntStream.range(1, 46)
```

Comment:
```suggestion
		IntStream.rangeClosed(1, 45)
```
이렇게도 표현할 수 있습니다!
그리고 1고 45는 상수로 관리해주면 어떨까요?